site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -471,6 +487,7 @@ The Id of a SCIM User Request Lambda that will be used to convert the SCIM Enter
 :enterprise_feature: SCIM
 include::../shared/_enterprise-edition-blurb-api.adoc[]
 +
+:enterprise_feature!:
```

Comment:
Did you add this for completeness or was it a doc bug?

Reply:
It was not causing display issues, but it seemed like the norm is to undefine that variable after each usage based on these type of comments:
```
// You must keep the empty line above the `+` and you cannot undefine :enterprise_feature!: here it breaks the formatting because of the list.
```

Daniel helped me find a way to format the `adoc` files that will allow using those includes and variables without the need for different spacing and variable usage. I could update the other cases in this file as well.

Thoughts?

Reply:
I looked at this again. The cases I updated were places that we could unset the variable without other changes.

I looked at updating other includes to be consistent, and they all behave differently. I did update the one remaining case that had the above comment with an unordered list, but I'm going to leave the others.